flybywiresim/a32nx
# fix(ISIS): LS scales diamond stays in scales and deviation is now the same as the PFD

Fixes #8040

## Summary of Changes


The scaling of the deviation indicator was way off and has now been made the same as on the PFD scales.
~~Additionally the magenta diamond is now constrained inside the scales even when the real deviation is way off.
This is according to V1 simulations the correct indication.~~

EDIT: According to new references the Diamond is actually only half visible when out of scale. 

## Screenshots (if necessary)



**PREVIOUS**
![PFD and ISIS LS mismatch](https://github.com/flybywiresim/a32nx/assets/5833783/daae2cee-ed9a-44af-ac8c-3d1e0c1443e6)

**NOW**
![IMG_4446](https://cdn.discordapp.com/attachments/1088616995144085574/1114530718811422730/image.png)


## References


V1 Simulations - A320 Capt

![IMG_4446](https://github.com/flybywiresim/a32nx/assets/5833783/4e618e09-1643-48ea-8ea1-ff7d3047a93e)




## Additional context



Discord username (if different from GitHub): Chaoz#3065

## Testing instructions


1. Get the aircraft ideally on a downwind to an ILS approach
2. Make sure that the ILS is tuned
3. Observe both the PFD and ISIS LS scales
4. When out of scale the ISIS magenta diamond should only show half at the edge of the scale just touching the tip against it
5. When inside scale both ISIS and PFD diamond position should be the same


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
